src/sparse_variational.jl

```diff
@@ -142,12 +300,19 @@ function AbstractGPs.elbo(
     ::SparseVariationalApproximation, ::FiniteGP, ::AbstractVector; kwargs...
 )
     return error(
-        "The observation noise fx.Σy must be homoscedastic.\n To avoid this error, construct fx using: f = GP(kernel); fx = f(x, σ²)",
+        "The observation noise fx.Σy must be homoscedastic.\n To avoid this error, ",
+        "construct fx using: f = GP(kernel); fx = f(x, σ²)",
```

Comment:
```suggestion
        "The observation noise fx.Σy must be homoscedastic.\nTo avoid this error, ",
        "construct fx using: f = GP(kernel); fx = f(x, σ²)",
```
or
```suggestion
        "The observation noise fx.Σy must be homoscedastic.\n",
        "To avoid this error, construct fx using: f = GP(kernel); fx = f(x, σ²)",
```
do you think it's sufficiently obvious here that σ² is meant to be a scalar?

Reply:
Hmm can't hurt to clarify. How about
```suggestion
        "The observation noise fx.Σy must be homoscedastic.\n",
        "To avoid this error, construct fx using: f = GP(kernel); fx = f(x, σ²)",
        ", where `σ²` is a positive `Real`.",
```
?

Reply:
a bit inconsistent with \` vs no \`. Given that the terminal doesn't actually convert \` into formatting, maybe just drop it...

Reply:
Good point.
```suggestion
        "The observation noise fx.Σy must be homoscedastic.\n",
        "To avoid this error, construct fx using: f = GP(kernel); fx = f(x, σ²)",
        ", where `σ²` is a positive Real.",
```
?

Reply:
I might have put the comma & space on the previous line still instead of at the start of a string :smile: but fine with me either way